### ISSUE TITLE
How to use this library without Fragment/Activity?

### ISSUE THREAD

**Issue by gab07** [jiangdongguo/AndroidUSBCamera#628]
I wonder if it is still possible on v3 to use this library without a fragment/activity like it used to work before.

**Comment by MattWanjia** [comment]
Hi did you find a solution to this?

**Comment by gab07** [comment]
Yes. Here is an example: https://github.com/gab07/opentok-react-native/blob/uvc-camera-update/android/src/main/kotlin/UvcVideoCapturer.kt
